GiniVision/Classes/GiniConfiguration.swift

```diff
@@ -184,6 +184,30 @@ import UIKit
      */
     public var cameraNotAuthorizedButtonTitleColor = UIColor.white
     
+    /**
+     Sets the background color of the new file import button tip
+     */
+    public var fileImportToolTipBackgroundColor = UIColor.white
+    
+    /**
+     Sets the text of the new file import button tip
+     */
+    public var fileImportToolTipText = "Du kannst jetzt auch ganz einfach Dateien hochladen."
+    
+    /**
+     Sets the text color of the new file import button tip
+     */
+    public var fileImportToolTipTextColor = UIColor.black
+    
+    /**
+     Sets the font of the new file import button tip
```

Comment:
Do you think users will know what "tip" means?

Reply:
Maybe _hint_ is more clear